flask_appbuilder/tests/security/test_password_complexity.py

```diff
@@ -0,0 +1,32 @@
+from flask_appbuilder.exceptions import PasswordComplexityValidationError
+from flask_appbuilder.validators import default_password_complexity
+
+from ..base import FABTestCase
+
+
+class PasswordComplexityTestCase(FABTestCase):
```

Comment:
Suggestion: Maybe you could use https://pypi.org/project/parameterized/ here?

Reply:
And few other places.

Reply:
did not know that one, only this: https://docs.pytest.org/en/6.2.x/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions.

Reply:
added looks better and it encourages to write more test cases (which I did) :)